app/src/main/java/com/example/vybe/LocationController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.vybe;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.location.Location;
+import android.location.LocationManager;
+import android.widget.Toast;
+
+import androidx.core.content.ContextCompat;
+
+public class LocationController {
+
```

Comment:
Is there anything else from other Location-related classes that can be moved into this controller? Seems kind of pointless to just have one method in here.